calico_kubernetes/calico_kubernetes.py

```diff
@@ -141,8 +158,119 @@ def _configure_interface(self):
         print('Finished configuring network interface')
         return ep
 
+    def _container_add(self, container_name, workload_id, pid, ip, interface,
```

Comment:
There methods were called "container_add" and "container_remove" in calicoctl / dockerland.  However, for kubernetes, I think we want to call these something pod-centric.

e.g. `add_pod(...)` and `remove_pod(...)`